apps/web/src/design-system/table/Table.tsx

```diff
@@ -90,7 +77,8 @@ export function Table({
   } = useTable(
     {
       columns,
-      data,
+      defaultColumn,
+      data: loading ? fakeData : data,
```

Comment:
When the table is in the loading state we will render the fake data and pass the `isLoading` prop to each cell, this way we will know when to show the loading skeleton and when to show the data.